service/matching/matchingEngine_test.go

```diff
@@ -1618,9 +1623,6 @@ func (s *matchingEngineSuite) TestTaskExpiryAndCompletion() {
 	const rangeSize = 10
 	s.matchingEngine.config.RangeSize = rangeSize
 	s.matchingEngine.config.MaxTaskDeleteBatchSize = dynamicconfig.GetIntPropertyFilteredByTaskQueueInfo(2)
-	// set idle timer check to a really small value to assert that we don't accidentally drop tasks while blocking
-	// on enqueuing a task to task buffer
-	s.matchingEngine.config.IdleTaskqueueCheckInterval = dynamicconfig.GetDurationPropertyFnFilteredByTaskQueueInfo(time.Millisecond)
```

Comment:
This cost me multiple days.

- The comment is non-sensical
- There is a bug in liveness where it treats the check interval as the TTL, so the effect here is that if a taskQueueManager is idle for a millisecond (which happens in some runs of this test), the liveness checker will kill it.
- The time value was recently changed from a microsecond to a millisecond, suggesting that it's just fiddly to begin with

In conclusion, I don't think setting this makes a meaningful difference to the test. I could leave it here set to 1*time.Second but what would be the point?